doc/modules/classes.rst

```diff
@@ -1486,6 +1486,24 @@ Utilities from joblib:
    utils.parallel_backend
    utils.register_parallel_backend
 
+.. _experimental_ref:
+
+Experimental
+============
+
```

Comment:
Let's note the different backwards compatibility policy here?

Reply:
done